index.html

```diff
@@ -7724,6 +7731,429 @@
                 </pre>
     </section>
   </section>
+    <section id="rtcquictransport*">
+    <h2>The RTCQuicTransport Object</h2>
+    <p>The <dfn><code>RTCQuicTransport</code></dfn> object includes information relating
```

Comment:
Do you plan to make it possible for RTCDataChannels to also be used with RTCQuicTransport, or just RTP?